index.ts

```diff
@@ -286,10 +286,22 @@ TEST: addTests('isQuickPR', [
 	'https://github.com/sindresorhus/refined-github/compare/test-branch?quick_pull=1',
 ]);
 
-export const isDraftPR = (): boolean => exists('#partial-discussion-header .octicon-git-pull-request-draft');
-export const isOpenPR = (): boolean => exists('#partial-discussion-header :is(.octicon-git-pull-request, .octicon-git-pull-request-draft)');
-export const isMergedPR = (): boolean => exists('#partial-discussion-header .octicon-git-merge');
-export const isClosedPR = (): boolean => exists('#partial-discussion-header :is(.octicon-git-pull-request-closed, .octicon-git-merge)');
+export const isDraftPR = (): boolean => ($('.State') ?? $('[class^="StateLabel"]'))!.textContent!.trim() === 'Draft';
+export const isOpenPR = (): boolean => {
+	const status = ($('.State') ?? $('[class^="StateLabel"]'))!.textContent!.trim();
```

Comment:
Why not join the selector in a single call?

Reply:
😮 New knowledge come into my head.